static/js/app.js

```diff
@@ -532,5 +532,8 @@ function dismissSurvey() {
 	survey_banner.classList.toggle("is-hidden");
 };
 
-document.getElementById("survey-dismiss").addEventListener("click", dismissSurvey, false);
+if ( document.getElementById("survey-dismiss") ) {
+  document.getElementById("survey-dismiss").addEventListener("click", dismissSurvey, false);
+}
+
```

Comment:
This was throwing a JS error on pages where the micro-survey-banner did not exist.